packages/scanner/src/cli/scan/watchScan.ts

```diff
@@ -139,11 +141,18 @@ export class Watcher {
 
     const scanner = await buildScanner(true, this.config, [appmapFile]);
 
+    const startTime = Date.now();
+    Telemetry.sendEvent({
+      name: 'watchscan:started',
+    });
```

Comment:
I'm glad that we are adding telemetry to `scan --watch`, but we are quickly going to regret sending a telemetry event for every single AppMap that is scanned. 

I would suggest that in `--watch` mode, telemetry should be batched up and sent periodically after there has been no activity for a defined period of time - like maybe 5 seconds - plus telemetry should be dumped on exit.

Reply:
By dumped on exit do you mean sent, or not sent?

Reply:
Moved to the entry point.